d2l-localize-behavior.js

```diff
@@ -167,5 +171,53 @@ D2L.PolymerBehaviors.LocalizeBehaviorImpl = {
 /** @polymerBehavior */
 D2L.PolymerBehaviors.LocalizeBehavior = [
 	AppLocalizeBehavior,
-	D2L.PolymerBehaviors.LocalizeBehaviorImpl
+	D2L.PolymerBehaviors.LocalizeBehaviorImpl,
+	{
```

Comment:
I was originally creating a new behavior that extended the original. This can probably just be merged with `LocalizeBehaviorImpl`